contracts/cw20-ics20/src/ibc.rs

```diff
@@ -399,7 +413,9 @@ mod test {
             msg: to_binary(&msg).unwrap(),
             funds: vec![],
         };
-        SubMsg::reply_on_error(exec, SEND_TOKEN_ID)
+        let mut msg = SubMsg::reply_on_error(exec, SEND_TOKEN_ID);
```

Comment:
Suggestion: Leet's add a `gas_limit` param to the `reply_on` fns for ergonomics.

Reply:
We do, but it takes a non-option (u64) there.
Which makes sense usually when you know you want to set limit to eg. 100.000, but not here

Reply:
Ah, I see there's a `with_gas_limit()`. Perhaps removing that and taking an `Option<u64>` `gas_limit` in these instead?

Reply:
It's in cosmwasm_std... 🤷‍♂️